### ISSUE TITLE
Capitalisation of UPosTag in annotated corpora

### ISSUE THREAD

**Issue by TomazErjavec** [clarin-eric/ParlaMint#64]
In the annotated corpora, the UD PoS tag is written in as the first element of the space separated value of `@ana`, as in 
https://github.com/clarin-eric/ParlaMint/blob/ffee99c04264c56e61383658ff22975102205602/ParlaMint-SI/ParlaMint-SI_2014-08-25_SDZ7-Izredna-01.ana.xml#L129

However, CZ capitalises it differently, as "UposTag", cf.
https://github.com/clarin-eric/ParlaMint/blob/ffee99c04264c56e61383658ff22975102205602/ParlaMint-CZ/ParlaMint-CZ_2014-09-10-ps2013-016-01-000-000.ana.xml#L144

 So @matyaskopp (and any others who have wrong capitalisation) could you pls. change it to "UPosTag".
And, I guess, this is something else that the validation should check.

**Comment by matyaskopp** [comment]
@TomazErjavec  I have taken it from here when I started working on our ParCzech project:
https://clarin-eric.github.io/parla-clarin/#sec-anawords

So is there any reason to do it in a different way than is in parla-clarin recommendation?

**Comment by TomazErjavec** [comment]
> @TomazErjavec I have taken it from here when I started working on our ParCzech project:
https://clarin-eric.github.io/parla-clarin/#sec-anawords

> So is there any reason to do it in a different way than is in parla-clarin recommendation?

Yes, my fault, sorry: when we started ParlaMint we (i.e. I) didn't take into account my own recommendations, so V1 use "UPosTag", and taking that as the example, others (except for you:) just looked at our existing examples. So, now we have "UPosTag" everywhere. I would ask that they change it back except that "UPosTag" is indeed more logical. 

I've now fixed https://clarin-eric.github.io/parla-clarin/#sec-anawords and pls. take into account the new recommendations, and, again, sorry for this blunder.
